account-kit/react/src/experimental/hooks/useSendCalls.ts

```diff
@@ -63,6 +63,11 @@ export type UseSendCallsResult<
  * or fall back to regular EOA transactions when connected to an EOA wallet. It handles the complete
  * flow of preparing, signing, and sending calls.
  *
+ * <Note>
+ * If using this hook with an ERC20 paymaster in pre-operation mode with `autoPermit`, the contents of the permit will be hidden
```

Comment:
Note for a different PR: our docs are inconsistent between "ERC20" and "ERC-20" and use both in a lot of places. We should standardize.